src/wallet.ts

```diff
@@ -359,15 +361,16 @@ export class Wallet {
     receiverId: string,
     contractName: string
   ): Promise<ResponseData<boolean>> {
-    const account = this.activeWallet?.account()
-    const accountId = this.activeWallet?.account().accountId
+    const accountId = this.getActiveAccountId()
 
-    if (!account || !accountId)
-      return formatResponse({ error: 'Account is undefined.' })
+    if (!accountId)
+      return formatResponse({ error: 'AccountId is undefined.' })
+    if (!this.activeAccount)
+      return formatResponse({ error: 'No activeAccount set.' })
     if (!contractName)
-      return formatResponse({ error: 'No contract was provided.' })
+      return formatResponse({ error: 'No contract name was provided.' })
 
-    const contract = new Contract(account, contractName, {
+    const contract = new Contract(this.activeAccount, contractName, {
```

Comment:
Every method that instantiates the contract with `this.activeAccount` errors out when called on the browser.

We were using `this.activeWallet?.account()` before.

Reply:
Should there be isNode/isBrowser logic to use `this.activeAccount` or `this.activeWallet?.account()`, respectively?

Reply:
bump :)

Reply:
> Should there be isNode/isBrowser logic to use `this.activeAccount` or `this.activeWallet?.account()`, respectively?

Yes, I think that makes sense yes :)

Reply:
@RNHTTR are you having any issues?

Reply:
Hey, sorry! I've been a bit busy. I plan to try and implement the isNode/isBrowser logic hopefully this week.

Reply:
No worries! Excited to take a look at it

Reply:
@microchipgnu hey -- i'm not using mintbase anymore and haven't had a chance to get back around to this. I'd like to, but I'm not sure when. If you want to make the update instead, please feel free :)